specification/src/main/asciidoc/platform/RelatedDocuments.adoc

```diff
@@ -4,149 +4,85 @@ This specification refers to the following
 documents. The terms used to refer to the documents in this
 specification are included in parentheses.
 
- _Java™ Platform, Enterprise Edition
-Specification Version 8_ (this specification). Available at
-_http://jcp.org/en/jsr/detail?id=366_ .
+Jakarta™ EE Platform Specification Version 8. Available at: <link needed>.
 
- _Java™ Platform, Standard Edition, v8 API
-Specification_ (Java SE specification). Available at
-_http://docs.oracle.com/javase/8/docs/[]http://docs.oracle.com/javase/8/docs/._
+Java™ Platform, Standard Edition, v8 API Specification (Java SE specification). Available at: http://docs.oracle.com/javase/8/docs/.
 
- _Enterprise JavaBeans™ Specification, Version
-3.2_ (EJB specification). Available at
-_http://jcp.org/en/jsr/detail?id=345_ .
+Jakarta™ Enterprise Beans Specification, Version 3.2. Available at: <link needed>.
 
- _JavaServer Pages™ Specification, Version 2.3_
-(JSP specification). Available at _http://jcp.org/en/jsr/detail?id=245_
-.
+Jakarta™ Server Pages Specification, Version 2.3. Available at: <link needed>.
 
- _Expression Language Specification, Version
-3.0_ (EL specification). Available at
-_http://jcp.org/en/jsr/detail?id=341_ .
+Jakarta™ Expression Language Specification, Version 3.0. Available at: <link needed>.
 
- _Common Annotations for the Java Platform
-Specification 1.3_ . Available at _http://jcp.org/en/jsr/detail?id=250_
-.
+Jakarta™ Annotations Specification, Version 1.3. Available at: <link needed>.
 
- _Java™ Servlet Specification, Version 4.0_
-(Servlet specification). Available at
-_http://jcp.org/en/jsr/detail?id=369_ .
+Jakarta™ Servlet Specification, Version 4.0. Available at: <link needed>.
 
- _JDBC™ 4.2 API_ (JDBC specification). Available
-at _http://jcp.org/en/jsr/detail?id=221_ .
+JDBC™ 4.2 API (JDBC specification). Available at: http://jcp.org/en/jsr/detail?id=221
```

Comment:
Again, don't remove the italics.   This is through out the PR, so I won't repeat myself.  :-)